### PR TITLE
add clever-tools tcp-redirs documentation

### PR DESCRIPTION
**Pull request by Keruspe**
There should probably be links somewhere towards this page

### CODE REVIEW

File: commons/clever-tools/tcp-redirections.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    TCP redirections are currently available free of charge as long as the feature is still considered in a BETA stage.
+    Once the feature leaves the BETA stage, it won't be free anymore.
+    </p>
+  </div>
+</div>
+
+## What is a TCP redirection?
+
+A regular application on Clever Cloud needs to listen on port 8080 and answer to HTTP traffic.
+Our reverse proxies then make your application available using HTTP(s) using you domain name.
```
Comment (urcadox) [inline commons/clever-tools/tcp-redirections.md:27]:
```suggestion
Our reverse proxies then make your application available using HTTP(s) using your domain name.
```

File: commons/clever-tools/tcp-redirections.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Some applications might additionally want to use a binary protocol, with raw TCP interactions.
+For this scenario, your application will have to listen for TCP traffic on port 4040.
+Once this is done, all you need to do is add a TCP redirection to your application. A port will be
+provided to you, and your TCP listener will be available on your domain name using the provided port.
+
+## What is a namespace?
+
+When creating a TCP redirection, you will need to chose in which namespace you want to create it.
```
Comment (urcadox) [inline commons/clever-tools/tcp-redirections.md:36]:
```suggestion
When creating a TCP redirection, you will need to choose in which namespace you want to create it.
```

File: commons/clever-tools/tcp-redirections.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+If you are a premium customer with dedicated reverse proxies, you will have your own namespace.
```
Comment (urcadox) [inline commons/clever-tools/tcp-redirections.md:37]:
```suggestion
If you are a Premium customer with dedicated reverse proxies, you will have your own namespace.
```

File: commons/clever-tools/tcp-redirections.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+If you are not a premium customer and want to use a custom domain name, you will have to use the
```
Comment (urcadox) [inline commons/clever-tools/tcp-redirections.md:38]:
```suggestion
If you are not a Premium customer and want to use a custom domain name, you will have to use the
```

File: commons/clever-tools/tcp-redirections.md (hunk 1 above)
Comment (hsablonniere) [inline commons/clever-tools/tcp-redirections.md:27]:
```suggestion
Our reverse proxies then make your application available using HTTP(s) using your domain name.
```

**Comment by Keruspe** [comment]
@hsablonniere I usually use inline when the command is part of a statement, and block when writing an actual standalone example

**Pull request merged**